src/services/queryRunner.ts

```diff
@@ -0,0 +1,41 @@
+import { sceneGraph, SceneObject, SceneQueryRunner } from '@grafana/scenes';
+
+import { FieldValuesBreakdownScene } from '../Components/ServiceScene/Breakdowns/FieldValuesBreakdownScene';
+import { LabelValuesBreakdownScene } from '../Components/ServiceScene/Breakdowns/LabelValuesBreakdownScene';
+import { logger } from './logger';
+import { interpolateExpression } from './query';
+import { getQueryRunnerFromChildren } from './scenes';
+
+export function getQueryRunnerFromSceneGraph(sceneRef: SceneObject) {
```

Comment:
Should we note somewhere that this is for PanelMenus

Reply:
It's also called in the FieldsAggregatedBreakdownScene